sdk/common/src/main/java/com/mesosphere/sdk/specification/GoalState.java

```diff
@@ -17,5 +17,10 @@
     /**
      * Running and then exiting successfully. Should not be restarted upon a successful exit.
      */
-    FINISHED
+    FINISH,
+
+    /**
+     * Running only once over the lifetime of a service. Should not be run again for the duration of this install.
```

Comment:
`Should not be run again ... unless indicated by an external agent e.g. end-user.`  Something like that.

Reply:
Doesn't that describe the `FINISH` state? What input would I provide to re-run a `ONCE` task?

Reply:
`dcos <service> plan restart <deploy> <phase> <step>`